apps/app/package.json

```diff
@@ -6,7 +6,6 @@
   "homepage": "http://devlaunchers.org/",
   "dependencies": {
     "@builder.io/partytown": "0.7.4",
-    "@cloudflare/wrangler": "^1.19.12",
```

Comment:
@chungthuang I removed `@cloudflare/wrangler` from the dependencies because it's deprecated and makes my github action fails. is this not used anywhere else?

Reply:
It's not used because we don't deploy the website as Cloudflare Workers.